### ISSUE TITLE
[dev.icinga.com #6164] Highlight focus to improve keyboard navigation

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#641]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/6164

**Created by tgelf on 2014-05-09 08:57:56 +00:00**

Assignee: _mjentsch_
Status: _Resolved_  (closed on _2014-12-30 15:40:02 +00:00_)
Target Version: _2.0.0-beta2_
Last Update: _2014-12-30 15:40:02 +00:00 (in Redmine)_

---

For design reasons many automagic outlines have been disabled. Keyboard navigation works fine for large parts of Icinga Web, but you are unable to see which element is currently focused.

Cheers,
Thomas

**Changesets**

_2014-05-09 16:09:03 +00:00_ by _tgelf_ 7153e41adaa216a0f2e99ea32d81d05a8c476289

    CSS: Highlight a couple of elements on focus
    
    Menu, tabs, hover menu and dropdown tabs. Keyboard navigation starts
    making sense right now.
    
    refs #6164

_2014-12-30 15:35:42 +00:00_ by _mjentsch_ e47eb73499880959564e05f45cb94c45bd4181b9

    Improve visibility of focus
    
    Display navigation focus in a visible color and display focus outline in regular site body more prominently.
    
    fixes #6164
